packages/uicore/src/components/FormikForm/FormikForm.tsx

```diff
@@ -163,9 +165,9 @@ function KVTagInput(props: KVTagInputProps & FormikContextProps<any>) {
       <BPTagInput
         values={
           isArray
-            ? formik?.values[name] || []
-            : Object.keys(formik?.values[name] || {}).map(key => {
-                const value = formik?.values[name][key]
+            ? fieldValue || []
+            : Object.keys(fieldValue || {}).map(key => {
```

Comment:
Is there any posibility of **fieldValue** to be string or number? If yes. This will break.

Reply:
It will always be a key value pair or array of key value pair